e2e/features/manage-fee-update.feature

```diff
@@ -119,21 +119,131 @@ Feature: Manage fee update
     Then I click the `Appeal` tab
     And I should see the text `Fee update details`
     And I should see `Decision type changed` for the `Reason for fee update` field
-    And I should see `£10.00` for the `New fee amount (£)` field
+    And I should see `£10.00` for the `New fee amount` field
     And I should see `Fee update recorded` for the `Completed stages` field
 
     When I switch to be a `Admin Officer`
     Then I click the `Appeal` tab
     And I should see the text `Fee update details`
     And I should see `Decision type changed` for the `Reason for fee update` field
-    And I should see `£10.00` for the `New fee amount (£)` field
+    And I should see `£10.00` for the `New fee amount` field
     And I should see `Fee update recorded` for the `Completed stages` field
 
     When I switch to be a `Judge`
     Then I click the `Appeal` tab
     And I should see the text `Fee update details`
     And I should see `Decision type changed` for the `Reason for fee update` field
-    And I should see `£10.00` for the `New fee amount (£)` field
+    And I should see `£10.00` for the `New fee amount` field
+    And I should see `Fee update recorded` for the `Completed stages` field
+
+  @Case-officer-manage-fee-update-appeal-not-valid @RIA-3793
```

Comment:
The e2e test should check if midevent is called and error is displayed for 'Appeal not valid' and fee amount >0
Please confirm

Reply:
Updated the tests for £0